### ISSUE TITLE
Keyboard Nav Alert dialog

### ISSUE THREAD

**Issue by Jaqster** [microsoft/pxt-microbit#6263]
We should have a nicer title for this dialog than "Alert".  Maybe "Need help with keyboard navigation?"
And it should have the focus be on the Ok button.
Better yet, a direct way to get to Help from this dialog.

![Image](https://github.com/user-attachments/assets/1a05b828-3a95-4192-b6a6-4d477f47c972)

**Comment by microbit-matt-hillsdon** [comment]
This is replaced by a toast in the latest version of the plugin (not yet release, hoping for something next week).

You can try a preview here with latest plugin and blockly code changes: https://preview.review-pxt.pages.dev/#editor

We need to chat about toast integration pros and cons on the next dev call.

**Comment by abchatra** [comment]
This is replaced by the toast which is live today.
